pkg/synopsysctl/cmd_update.go

```diff
@@ -604,9 +604,14 @@ var updateBlackDuckAddRegistryCmd = &cobra.Command{
 		if err != nil {
```

Comment:
Where is namespace set in the above getInstnaceInfo call?

Reply:
https://github.com/blackducksoftware/synopsys-operator/pull/514/files#diff-dbf802f9b8914536b969f1a6555170d4L934 The flag is Persistent so it's passed down to the sub commands.

Reply:
Gotcha